cardano-cli/src/Cardano/CLI/EraBased/Commands/Governance/DRep.hs

```diff
@@ -67,7 +73,13 @@ data GovernanceDRepUpdateCertificateCmdArgs era
   = GovernanceDRepUpdateCertificateCmdArgs
   { eon :: !(ConwayEraOnwards era)
   , drepHashSource :: !DRepHashSource
-  , mAnchor :: !(Maybe (L.Anchor (L.EraCrypto (ShelleyLedgerEra era))))
+  , mPotentiallyCheckedAnchor
```

Comment:
I'd keep the field name `mAnchor` since the type already gives information about the need to check the hash. Long field names are hard to parse IMHO.

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/commit/773ad8cedb7b6f5676e6d7f373be15633fae5abd